src/Symfony/Component/Console/Formatter/OutputFormatter.php

```diff
@@ -34,15 +34,13 @@ public function __clone()
     }
 
     /**
-     * Escapes "<" special char in given text.
+     * Escapes "<" and ">" special chars in given text.
      *
-     * @param string $text Text to escape
```

Comment:
This line should be kept, right?

Reply:
Yes sorry I started with the 5.4 code and missed that when backporting, will fix in a bit

Reply:
OK I made sure all the unnecessary diffs are gone now